src/libs/runtime-utils.js

```diff
@@ -166,3 +166,16 @@ export const appIsSettingUp = app => {
 export const collapsedRuntimeStatus = runtime => {
   return runtime && (runtime.patchInProgress ? 'LeoReconfiguring' : runtime.status) // NOTE: preserves null vs undefined
 }
+
+export const convertedAppStatus = appStatus => {
```

Comment:
try `Utils.switchCase` for style consistency, 
ex: https://github.com/DataBiosphere/terra-ui/blob/dev/src/components/NewRuntimeModal.js#L1130

Reply:
Addressed and fixed